### ISSUE TITLE
access to the web interface only from one computer

### ISSUE THREAD

**Issue by Merschi68** [pauhull/MarlinrakerOS#2]
Hello,
after successful installation and configuration I can access the web interface in my network only from the PC from which I configured everything.
All other computers only show the Connecting page.
The used browser does not matter.
Also about the IP address as the web interface is not displayed.
According to the browser there are problems accessing websocket(index.d22c661a.js).

**Comment by phm07** [comment]
Hi,

this seems like a weird issue to me because it shouldn't matter from which computer you configured your Pi from.
You said the web interface shows up and is trying to connect? So there is a connection to your Pi?

You could check this by trying to SSH into your Pi from a computer where you can't access the web interface from. If it can't connect it's probably a networking or configuration issue and not a software issue.

If the page shows up but doesn't connect, can you make a full screenshot of the developer console and post it here? That would be helpful.

You could also try to enable the `extended_logs` option in the configuration file. This prints debug information to the log file, it would be helpful if you would attach that one too.

Best regards

**Comment by Merschi68** [comment]
Hello,
I think we can close the case.
I just used 2 other networks for testing and in both cases the connection works.
I suspect that at home a transparent proxy triggers the problem and will check there again.
By the way, I also did a new installation yesterday evening with the same problems at the end.

Thanks again for the quick reply.

Übersetzt mit DeepL (https://www.deepl.com/app/?utm_source=ios&utm_medium=app&utm_campaign=share-translation
